AudioDeviceExample/ViewController.swift

```diff
@@ -305,6 +305,7 @@ class ViewController: UIViewController {
     }
 
     func logMessage(messageText: String) {
+        NSLog(messageText)
```

Comment:
Can we use `print()` instead?

Reply:
We can, but then you don't have the timestamp. So when its interspersed with the logs out of the SDK, it looks a little bit weird. I just copied out what we had in the other example apps because I wanted to see it in the console as I was working through this. If we change it here, we should change it in all the implementations of `logMessage`.

Reply:
I guess I have a bit of aversion to NSLog in Swift code, but I see your point about the timestamps. I can go either way, @piyushtank can you break the tie?